Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -142,6 +142,8 @@ def descfile(font):
 @condition
 def description(descfile):
   """Get the contents of the DESCRIPTION file of a font project."""
+  if not descfile:
+    return
```

Comment:
I thought that a condition should fail by default if a condition of the condition was not satisfied at first.